router/batchrouter/batchrouter.go

```diff
@@ -814,9 +814,12 @@ func (brt *HandleT) asyncUploadWorker(ctx context.Context) {
 
 				timeElapsed := time.Since(brt.asyncDestinationStruct[destinationID].CreatedAt)
 				brt.asyncDestinationStruct[destinationID].UploadMutex.Lock()
-				if brt.asyncDestinationStruct[destinationID].Exists && (brt.asyncDestinationStruct[destinationID].CanUpload || timeElapsed > brt.asyncUploadTimeout) {
+				uploadInterval, _ := destinationsMap[destinationID].Destination.Config["uploadInterval"].(string)
```

Comment:
What if `destinationsMap[destinationID].Destination.Config` is missing or not a string ?

Can we do some error handling or fallback to default along with a warning?

Reply:
the fallback is in place:
in case it's missing/not a string -> `uploadInterval` turns out to be an empty string -> falls back to default(`brt.asyncUploadTimeout`) in `parseTime`
I'll add a warning log statement.

Reply:
Actually, the default time is used but is not implicit. We are passing an empty string, which results in a parsing error leading to the default value.
https://go.dev/play/p/bVucagUngy9

Reply:
> the fallback is in place:
 My bad I understood it later, but it makes it a bit harder to read if the errorhandling happens implicitly somewhere else.

Reply:
Correct. I'll add a log and fall back even before we get to calling `parseTime`